molgenis_documentation/user_documentation.md

```diff
@@ -947,6 +951,163 @@ To get the feel of how certain styles look, you can select it in the dropdown me
 
 For an overview of all the different themes offered, visit the [Bootswatch](https://bootswatch.com/) website. It is currently not yet possible to submit your own CSS sheets to update the styling. We do however plan to implement this in the future, giving you even more control to add your own personal style to your MOLGENIS research database.
 
+## <a name="molgenis-modules"></a>MOLGENIS modules
+### <a name="upload"></a>Upload
+The upload module is the place in MOLGENIS where you can upload your data into the MOLGENIS application.
+
+#### <a name="upload-possibilities"></a>Upload possibilities
+The MOLGENIS upload module supports the following upload files formats and data:
+	
+|file format		|file extention             |data formats|
+||
+|CSV              	|".csv" ".txt" ".tsv" ".zip"|EMX|
+|Excel            	|".xls" ".xlsx"             |EMX|
+|OWL              	|".owl.zip"                 |OWL|
+|VCF (version 4.0)	|"vcf" ".vcf.gz"            |VVCF (version 4.0)|:
+
+##### <a name="upload-possibilities-abbreviations"></a>Abbreviations:
+
+* CSV: Comma Seperated Value
+* OWL: Web Ontology Language
+* VCF: Variant Call Format
+* EMX: Entity Model Extensible
+
+#### <a name="upload-screens"></a>Upload pages
+The different pages will be explianed by uploading the<a name="advanced-data-example"></a> "Advanced data example" ([download](download/advanced_data_example_v20151104.xlsx)) example data set.
+
+The pages are: 
+
+1. Upload file
+2. Options
+3. Packages
+4. Validation
+5. Result
+
+Navigation buttons at the bottom of the pages:
+
+* Previous: Go to the previous page.
+* Next: Go to the next page.
+* Restart: Push this button when you want to start importing a new data set. It will redirect you to the start of this wizard. Pushing this button will kill the session but not the importing job.
+* Finish: The same as Restart.
+
+
+##### <a name="upload-screens-upload-file"></a>Upload file page
+
+###### <a name="upload-screens-options-steps"></a>Steps
+
+1. Select a file to upload.
+2. Click on the next button.
+
+![Upload file screen](images/plugin-upload/upload-file-screen.png?raw=true, "Upload file")
+
+##### <a name="upload-screens-options"></a>Options page
+Select a data upload option. On this page you can select the rules of how to upload your data into MOLGENIS. Because this dataset is an unknown dataset to the application we leave the default option "Add entities" selected. In tabular data sets, the term entities refers to data-rows.
```

Comment:
Maybe use "new dataset" instead of "unknown dataset" ? not sure... Also the mixing of the terms dataset and entity might be confusing(UI components all use entity). Might be an option to use "entity" and add a line at the beginning that we mean the dataset by that?